org.tildearrow.furnace.yml

```diff
@@ -0,0 +1,79 @@
+app-id: org.tildearrow.furnace
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: furnace
+rename-desktop-file: furnace.desktop
+rename-appdata-file: furnace.appdata.xml
+rename-icon: furnace
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+
+modules:
+  - shared-modules/SDL2/SDL2-with-libdecor.json
```

Comment:
23.08 has this https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/1533f8d0393984f5684985c8b710e9fbaf9c8734/elements/components/sdl2.bst#L12, https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/1533f8d0393984f5684985c8b710e9fbaf9c8734/elements/components/sdl2.bst#L32